core/frontend/src/components/video-manager/VideoStreamCreationDialog.vue

```diff
@@ -47,11 +47,19 @@
             :key="index"
             class="d-flex justify-space-between align-center"
           >
+            <v-select
+              :items="[StreamType.UDP, StreamType.RTSP]"
+              :value="endpoint.startsWith('rtsp://') ? StreamType.RTSP : StreamType.UDP"
+              class="mr-10"
+              style="width:20%;"
+              @change="set_default_address_for_stream(index, $event)"
+            />
             <v-text-field
               v-model="stream_endpoints[index]"
               label="Stream endpoint"
               placeholder="e.g. udp://192.168.2.2:5600 or rtsp://0.0.0.0:8554/video0"
```

Comment:
```suggestion
              placeholder="e.g. udp://192.168.2.1:5600 or rtsp://192.168.2.2:8554/video0"
```

just to be more consistent with the default use case.

Reply:
we have access to the interface on the other side, too. I'll update.

I wish we could have the camera manager do it though... so we could use it on all interfaces. We can do that later